stable_baselines3/sac/sac.py

```diff
@@ -259,6 +267,11 @@ def learn(self,
                                      n_eval_episodes, eval_log_path, reset_num_timesteps)
         callback.on_training_start(locals(), globals())
 
+        if self.tensorboard_log is not None and SummaryWriter is not None:
```

Comment:
Looks like something that should be done in the base class, no? (to avoid code duplication)